src/EnergyPlus/HWBaseboardRadiator.cc

```diff
@@ -1724,7 +1719,7 @@ namespace HWBaseboardRadiator {
         int ZoneNum;              // Pointer to the Zone derived type
         Real64 ThisSurfIntensity; // temporary for W/m2 term for rad on a surface
 
-        auto &NumHWBaseboards = state.dataHWBaseboardRad->NumHWBaseboards;
+        int NumHWBaseboards = state.dataHWBaseboardRad->NumHWBaseboards;
         auto &HWBaseboard = state.dataHWBaseboardRad->HWBaseboard;
```

Comment:
const ref for HWBaseboard, HWBaseboardDesignObject, and QBBRadSource?

Reply:
These should actually be removed and replaced with references to individual array elements, e.g., `thisHWBaseboard`. Not going to do that in this PR because it will get too big and I won't be able to track down diffs.

Reply:
Good idea to address these in a separate PR